agenta-backend/agenta_backend/models/api/api_models.py

```diff
@@ -107,6 +114,9 @@ class AppVariantResponse(BaseModel):
     config_name: str
     uri: Optional[str]
     revision: int
+    date_modified: str
+    date_created: str
+    modified_by: ModifiedBy
```

Comment:
Could we simply add the `user_id` here ?

Reply:
Applicable wherever this happens.

Reply:
@bekossy, this has been updated to modified_by_id, so you’ll need to explicitly fetch the user who made the modification using this endpoint: `/api/profile?user_id={userId}`